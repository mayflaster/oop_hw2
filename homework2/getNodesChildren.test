CreateGraph A
ListChildren A n1

CreateNode n1 10
AddNode A n1
ListChildren A n1

CreateNode n2 20
ListChildren A n2
AddNode A n2
AddEdge A n1 n2
ListChildren A n1

AddEdge A n1 n1
AddEdge A n2 n2

ListChildren A n1
ListChildren A n2

CreateNode a1 50
CreateNode a2 60
AddNode A a1
AddNode A a2
AddEdge A n1 a2
AddEdge A n1 a1
ListChildren A n1
