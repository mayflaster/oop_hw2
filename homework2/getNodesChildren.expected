created graph A
node n1 does not exist

created node n1 with cost 10
added node n1 to A
the children of n1 in A are:

created node n2 with cost 20
failed to get n2 list of children in A: node does not exist
added node n2 to A
added edge from n1 to n2 in A
the children of n1 in A are: n2

added edge from n1 to n1 in A
added edge from n2 to n2 in A

the children of n1 in A are: n1 n2
the children of n2 in A are: n2

created node a1 with cost 50
created node a2 with cost 60
added node a1 to A
added node a2 to A
added edge from n1 to a2 in A
added edge from n1 to a1 in A
the children of n1 in A are: a1 a2 n1 n2
